config

```diff
@@ -0,0 +1,66 @@
+####################
```

Comment:
`.gitignore` is for ignoring non-tracked file.
There’s [no clean way to commit a file but to not track its changes](https://stackoverflow.com/questions/4633681/git-ignoring-version-controlled-files).

So, what I did is:
* commit a `config.sample` file that is tracked by git and should not be modified by users.
* The `config` file has to be created by the user (possibly by copying `config.sample`) and is ignored by git.
* `cluster.sh` uses `config` if it exists or invites the user to create one by copying `config.sample` if it doesn’t.